python/QC4_HV.py

```diff
@@ -0,0 +1,24 @@
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+args = parser.parse_args()
+cmd = ["python","/Users/anastasia/desktop/QC_Plotting_Tools/Produce_Config_File.py"]
```

Comment:
Do not use hard coded paths.

Use the environment variable $GEM_BASE.  Moreover why is this pointing to a directory **outside** the repository?  `Produce_Config_File.py` should be inside the repository for it to be tracked correctly.